packages/graphql-transformers-e2e-tests/src/__tests__/CustomRoots.e2e.test.ts

```diff
@@ -128,6 +128,11 @@ test('Test custom root query, mutation, and subscriptions.', () => {
 
         authedField: String
             @aws_auth(cognito_groups: ["Bloggers", "Readers"])
```

Comment:
I don't believe this test is actually deployed through CFN and I believe this would fail since the server has validation that does not allow @aws_auth and @aws_cognito_user_pools to be used at the same time. Can we move multi auth tests to their own e2e test file that actually deploys to CFN so we can test the full e2e flow?